core/templates/components/certificate-assessment-offering-helper/time-expired-modal.component.html

```
@@ -0,0 +1,46 @@
+<div class="assessment-modal-overlay">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)